### PR TITLE
Change Send button to Enter Icon, change the corresponding selector in test files

### PR DESCRIPTION
**Pull request by czhou29**
## Checklist

- [ ] Did you write tests for your code?
- [x] Did all the tests pass locally?

## Trello Ticket

https://trello.com/c/VwuOq7VB/79-change-send-button-to-an-enter-icon

## Short Description

Change Send button to Enter Icon, change the corresponding selector in test files

### CODE REVIEW

File: src/modules/ChatRoom.js [hunk 1]
```diff
@@ -367,18 +367,37 @@ class ChatRoom extends React.Component {
...
+                      width="1em"
+                      height="1em"
+                      viewBox="0 0 16 16"
+                      className="bi bi-arrow-return-left"
+                      fill="currentColor"
+                      xmlns="http://www.w3.org/2000/svg"
+                    >
+                      <path
+                        fillRule="evenodd"
+                        d="M14.5 1.5a.5.5 0 0 1 .5.5v4.8a2.5 2.5 0 0 1-2.5 2.5H2.707l3.347 3.346a.5.5 0 0 1-.708.708l-4.2-4.2a.5.5 0 0 1 0-.708l4-4a.5.5 0 1 1 .708.708L2.707 8.3H12.5A1.5 1.5 0 0 0 14 6.8V2a.5.5 0 0 1 .5-.5z"
```
Comment (Joyfal) [inline src/modules/ChatRoom.js:395]: Can you create a svg for this and import it here?

**Review by Joyfal — CHANGES_REQUESTED** [review]
Looks good, one small thing.

File: src/modules/ChatRoom.js [hunk 2]
```diff
@@ -367,18 +367,37 @@ class ChatRoom extends React.Component {
                   onChange={this.handleChange}
                   placeholder="Potatoes can't talk... but you can!"
                 />
-                {/* <div className="input-group-append">
-                      <button type="button" className="btn" onClick={() => this.toggleEmojiPicker()} id="show-emoji-yes">{'😃'}</button>
-                  </div> */}
+                <div className="input-group-append">
```
Comment (Joyfal) [inline src/modules/ChatRoom.js:370]: Please change the css to have chat meesage window use the addtional space (look at the empty space right above input box)

Comment (macedo22) [reply src/modules/ChatRoom.js:370]: Wow what an eagle eye you have

Comment (macedo22) [reply src/modules/ChatRoom.js:370]: Wow this was a good catch

File: src/modules/enter.svg [hunk 3]
```diff
@@ -0,0 +1,11 @@
+<svg
```
Comment (Joyfal) [inline src/modules/enter.svg:1]: Maybe move this svg file outside modules (to src/). modules should only have modules files, not svg.

**Review by Joyfal — CHANGES_REQUESTED** [review]
Almost there.

File: src/modules/enter.svg (hunk 3 above)
Comment (czhou29) [reply src/modules/enter.svg:1]: Okay

**Review by Joyfal — APPROVED** [review]
LGTM

**Pull request merged**